docs/provider-pkcs11.7.md

```diff
@@ -0,0 +1,262 @@
+% provider-pkcs11(7) | Configuration directives
+
+NAME
+====
+pkcs11-provider - An OpenSSL provider that allows to directly interface
+with pkcs11 drivers.
+
+
+DESCRIPTION
+===========
+
+Starting with version 3.0 the OpenSSL project introduced a new modular
+system to exten OpenSSL that replaces the deprecated Engines modules
+
+Providers[1] are loaded via configuration directives in the openssl
+configuration file (or via direct loading by applications).
+
+The pkcs11 provider allows openssl linked applications to use keys and
+cryptographic operations from a hardware or software token via their
+PKCS#11[2] driver and the use ok pkcs11 URIs[3].
+
+The pkcs11 provider can be configured to be automatically loaded via
+openssl.cnf
+
+
+CONFIGURATION
+=============
+Configuration options recognized by the provider
+
+## pkcs11-module-path
+
+A file path to the pkcs11 driver to be used
+
+Default: None (but see the environment variables section)
```

Comment:
Default is p11-kit pkcs11 proxy from (`DEFAULT_PKCS11_MODULE`) if built with p11-kit:

https://github.com/latchset/pkcs11-provider/blob/874aa56f56d815c2f64406999a5df46f45110deb/src/interface.c#L234